docs/config.md

```diff
@@ -514,6 +514,28 @@ conflict is considered fully resolved when there are no conflict markers left.
 
 ## Git settings
 
+### Default remotes for `jj git fetch` and `jj git push`
+
+By default the remote used for `jj git fetch` and `jj git push` is the remote
+called `"origin"`, just like in Git. Sometimes this is undesirable, e.g. when
```

Comment:
nit: If there's only one remote, we use that for fetch/push. If there are multiple, we use "origin" by default